websites/B/Brain.fm/dist/metadata.json

```
@@ -0,0 +1,32 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dzlandis",
+		"id": "259708781166985217"
+	},
+	"service": "Brain.fm",
+	"description": {
+		"en": "Brain.fm provides music designed for the brain (generated by an AI they've invented) to improve focus, meditation, relaxation, naps & sleep within 15 minutes of use."
```

This is not the same as the description from the metadata
```suggestion
		"en": "Use Brain.fm's functional music to help you focus relax and sleep better, using cutting edge neuroscience to get results."
```